### PR TITLE
how-to: update ansible document for v4.0

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update Ansible deployment and upgrade documents for v4.0 (only apply to 4.0, does not apply to master).

The changes in this PR is migrated from https://github.com/pingcap/docs-cn/pull/2583.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/2583

### CODE REVIEW

File: how-to/upgrade/from-previous-version.md [hunk 1]
```diff
@@ -72,20 +68,20 @@ Version: 0.9.0
...
+以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录，备份 TiDB 2.0、2.1、3.0、3.1 或其他低版本的 tidb-ansible 文件夹：
 
 {{< copyable "shell-regular" >}}
 
 ```bash
 mv tidb-ansible tidb-ansible-bak
 ```
 
-下载 TiDB latest 版本对应的 tidb-ansible  [**下载 TiDB Ansible**](/how-to/deploy/orchestrated/ansible.md#在中控机器上下载-tidb-ansible)，默认的文件夹名称为 `tidb-ansible`。
+下载 TiDB 4.0 版本对应的 tidb-ansible  [**下载 TiDB Ansible**](/how-to/deploy/orchestrated/ansible.md#在中控机器上下载-tidb-ansible)，默认的文件夹名称为 `tidb-ansible`。
```
Comment (lilin90) [inline how-to/upgrade/from-previous-version.md:79]:
```suggestion
[**下载 TiDB 4.0 版本对应的 TiDB Ansible**](/how-to/deploy/orchestrated/ansible.md#在中控机器上下载-tidb-ansible)，默认的文件夹名称为 `tidb-ansible`。`$tag` 需替换为选定的 TAG 版本的值，例如 `v4.0.0-beta.2`。
```

File: how-to/upgrade/from-previous-version.md [hunk 2]
```diff
@@ -169,13 +165,13 @@ git clone  https://github.com/pingcap/tidb-ansible.git
 
     > **注意：**
     >
-    > 最新开发版单机多 TiKV 实例（进程）情况下，需要添加 `tikv_status_port` 参数。
+    > 从 3.0 以前版本升级到 4.0 版本，并且单机多 TiKV 实例（进程）情况下，需要添加 `tikv_status_port` 参数。
```
Comment (lilin90) [inline how-to/upgrade/from-previous-version.md:168]: 这里包括 3.0 版本吗，是 < 3.0 还是 ≤ 3.0 呢？

Comment (liubo0127) [reply how-to/upgrade/from-previous-version.md:168]: 不包括 3.0，< 3.0

File: how-to/upgrade/from-previous-version.md (hunk 1 above)
Comment (liubo0127) [reply how-to/upgrade/from-previous-version.md:79]: LGTM

**Comment by TomShawn** [comment]
@lilin90 Suggestions are committed, PTAL.

**Pull request merged**